src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -8,6 +8,7 @@
     "CdkProjectTemplate": "../CdkTemplates/AspNetAppElasticBeanstalkLinux",
     "CdkProjectTemplateId": "netdeploy.AspNetAppElasticBeanstalkLinux",
     "Description": "Deploy an ASP.NET Core application to AWS Elastic Beanstalk. Recommended for applications that are not set up to be deployed as containers.",
+    "ShortDescription": "Deploy an ASP.NET Core application to AWS Elastic Beanstalk.",
```

Comment:
Suggested text:

"ASP.NET Core application deployed to AWS Elastic Beanstalk on Linux."

Reply:
Fixed